arm/Microsoft.Automation/automationAccounts/.parameters/parameters.json

```diff
@@ -125,6 +155,15 @@
         //         }
         //     ]
         // },
+        "systemAssigned": {
+            "value": true
+        },
+        "userAssignedIdentities": {
+            "value": {
+                "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-001": {},
+                "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/sxx-az-msi-x-001": {}
```

Comment:
please remove sxx-az-msi-x-001 as it should ideally be deleted as part of its module

Reply:
Done